packages/exclusions/src/types/index.ts

```diff
@@ -50,6 +51,10 @@ export class ExclusionsRulesDescriptor {
             }
         }
 
+        if (rawData.hasOwnProperty('modified') && typeof (rawData.modified) === 'object') {
```

Comment:
I think you can just do `rawData?.modified === 'object'`

Reply:
I already made the change
typeof (rawData?.modified) === 'object'